index.css

```diff
@@ -262,5 +279,261 @@ a {
+    flex-wrap: wrap;
+}
+
+.photo-row {
+    max-height: 355px;
+    max-width: 645px;
+}
+
+.photo-row-line-one {
+    z-index: 30;
+    position: relative;
+}
+
+.photo-row-line-one::before {
+    content: '';
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background-color: #15182460;
+    background-blend-mode: multiply;
+    z-index: 50;
+    position: absolute;
+    height: auto;
+    width: auto;
+}
+
+.photo-row-content {
+    z-index: 70;
+    position: absolute;
+    bottom: 28px;
+    margin-left: 32px;
+    color: #FFFFFF;
+    font-size: 50px;
+    font-weight: 600;
+}
+
+.photo-frame-second-line {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 10px;
+    flex-wrap: wrap;
+}
+
+.photo-row-line-two {
+    position: relative;
+    z-index: 30;
+}
+
+.photo-row-line-two::before {
+    content: '';
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background-color: #15182460;
+    background-blend-mode: multiply;
+    z-index: 50;
+    position: absolute;
+    height: auto;
+    width: auto;
+}
+
+.photo-row2 {
+    max-height: 283px;
+    width: 414px;
+}
+
+/* -------------- LOCATION DESCRIPTIONS -------------- */
+
+    .location-description-container {
+        margin-top: 131px;
+    }
+
+    .location-description {
+        display: flex;
+        justify-content: space-around;
+        margin-bottom: 78px;
+    }
+
+    .photo-location {
+        max-width: 597px;
+    }
+
+    .location-content {
+        padding: 37px 0 70px 111px;
+    }
+
+    .subheader-location {
+        color: #102746;
+        font-size: 30px;
+        font-weight: 600;
+        line-height: 40px;
+    }
+
+    .blog-content-container {
+        padding: 70px 0;
+    }
+
+    .second-photo {
+        order: 2;
+    }
+
+    .second-content {
+        order: 1;
+        padding: 37px 111px 70px 0;
+    }
+
+    /* -------------- PLACES TO VISIT - SOCIAL -------------- */
+        .places-visit {
+            height: 1373px;
+            position: relative;
+        }
+
+        .picture-location-table {
+            display: flex;
+            justify-content: space-evenly;
+            margin-bottom: 58px;
+        }
+
+        .picture-location-description {
+            width: 392px;
+        }
+
+        .picture-location-description-column {
+            display: flex;
+            justify-content: space-evenly;
+            flex-direction: column;
+        }
+    
+        .sub-subtitle { /*styling to reuse for h3*/
+            font-weight: 400;
+            padding-bottom: 55px;
+            font-size: 35px;
+            color: #102746;
+        }
+
+        .social-bar-image {
+            z-index: 30;
+            position: relative;
+        }
+        .social-bar-image::before {
+            content: '';
+            top: 322px;
+            right: 0;
+            left: 0;
+            bottom: 0;
+            background-color: #15182460;
+            background-blend-mode: multiply;
```

Comment:
https://caniuse.com/#search=multiply ceci est encore experimental

Reply:
Mes notes la-dessus: 
"I suggested to Raff to increase the opacity of the dark overlay so that the copy wouldn't appear white-on-white. So we increased it to 60% instead of 40% like in the .sketch file. We also added a background-blend-mode: multipy to have the colors pop more."

Est-ce qu'il y aura une meilleure altermative? C'est quoi le best practice? Je le laisse?